### PR TITLE
[HOPSWORKS-2763] Opendistr 1 13 2

### PR DESCRIPTION
**Pull request by jimdowling**
https://logicalclocks.atlassian.net/browse/HOPSWORKS-2763

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -13,8 +12,9 @@
 #
 # Logstash
 #
-default['logstash']['version']                               = "7.2.0"
-default['logstash']['url']                                   = "#{node['download_url']}/logstash-oss-#{node['logstash']['version']}.tar.gz"
+default['logstash']['version']                               = "7.10.2"
+default['logstash']['url']                                   = "#{node['download_url']}/opendistro/logstash-#{node['logstash']['version']}-linux-x86_64.tar.gz"
```
[the remark this answers is not in the document]
Comment (jimdowling) [reply attributes/default.rb]: ok, thanks. upgraded filebeat.

File: attributes/default.rb [hunk 2]
```diff
@@ -13,8 +12,9 @@
...
+#default['logstash']['sha512']                   = "ddd6c0db8b59b9ad719a69f04eea81e0261061b5a26569819d4c9306c6f3a63acf575bf50e5f22b17e9b44ed8750d83762b41b866a25b2a2e283b5186c9cdf61"
```
[the remark this answers is not in the document]
Comment (jimdowling) [reply attributes/default.rb]: We're not validating the hashes, but i can added it anyway :)

File: attributes/default.rb [hunk 3]
```diff
@@ -58,8 +58,9 @@
 #
 # Filebeat
 
-default['filebeat']['version']                  = "7.2.0"
+default['filebeat']['version']                  = "7.10.2"
 default['filebeat']['url']                      = "#{node['download_url']}/filebeat-oss-#{node['filebeat']['version']}-linux-x86_64.tar.gz"
```
Comment (maismail) [inline attributes/default.rb:62]:
```suggestion
default['filebeat']['url']                      = "#{node['download_url']}/opendistro/filebeat-oss-#{node['filebeat']['version']}-linux-x86_64.tar.gz"
```

**Comment by jimdowling** [comment]
The upgrade to opensearch 1.1.0 appears to work - using the bundled JDK. Closing this PR, opening a new one.

**Pull request closed without merging**